frontend/amundsen_application/static/js/components/OwnerEditor/index.tsx

```diff
@@ -238,21 +241,19 @@ export class OwnerEditor extends React.Component<
     );
   };
 
-  render() {
-    const { isEditing, readOnly, resourceType } = this.props;
-    const { errorText, itemProps } = this.state;
-    const hasItems = Object.keys(itemProps).length > 0;
-
-    if (errorText) {
-      return (
-        <div className="owner-editor-component">
-          <span className="status-message">{errorText}</span>
-        </div>
-      );
-    }
+  renderOwnersSection = (section: OwnerCategory | null) => {
```

Comment:
we might want to consider doing some indication if there are no owners in a certain category. right now those titles look like floating text but isn't fully clear that it is an empty list

Reply:
updated -- i used the styling that FrequentUsers uses for "No frequent users exist": https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/static/js/pages/TableDetailPage/FrequentUsers/index.tsx#L64